src/Chapter1.hs

```diff
@@ -489,9 +490,10 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
 
+lastDigit :: Integral a => a -> a
+lastDigit n | n > 0 = mod n 10
+            | n < 0 = mod n (-10)
```

Comment:
Nice view on different conditions. However, when `n == 0` there is no such clause to cover that.

Also, instead of manually handling this, you can use `abs` function on `n` 🙂